### PR TITLE
Pioneer 18d updates

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Deployment times and location can be reviewed in P18d QLR (3204-01812) and sensor configurations can be reviewed in FCPs (3304-00150-00160 and 3304-00180-00008)

**Review by rtravis-whoi — APPROVED** [review]
approved deploy csvs; I created the NUTNR cal csv so I can't review that.

File: deployment/CP05MOAS-GL559_Deploy.csv [hunk 1]
```diff
@@ -22,4 +22,10 @@ TS20220714,,TS20220714,,CP05MOAS-GL559-01-ADCPAM000,4,1,2022-07-14T17:45:00,2022
...
+TS20220714,,TS20220714,,CP05MOAS-GL559-05-PARADM000,4,1,2022-07-14T17:45:00,2022-07-14T21:53:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50304,39.85742,-70.6229,,1000,1000,FZ-1,,
+TS20220822,,,,CP05MOAS-GL559-00-ENG000000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.83375,-70.37818,,1000,1000,FZ-1,,
+TS20220822,,,,CP05MOAS-GL559-01-ADCPAM000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.83375,-70.37818,,1000,1000,FZ-1,,
+TS20220822,,,,CP05MOAS-GL559-02-FLORTM000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-04201,39.83375,-70.37818,,1000,1000,FZ-1,,
+TS20220822,,,,CP05MOAS-GL559-03-CTDGVM000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09347,39.83375,-70.37818,,1000,1000,FZ-1,,
+TS20220822,,,,CP05MOAS-GL559-04-DOSTAM000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00195,39.83375,-70.37818,,1000,1000,FZ-1; top mount,,
+TS20220822,,,,CP05MOAS-GL559-05-PARADM000,5,1,2022-08-22T16:39:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50304,39.83375,-70.37818,,1000,1000,FZ-1,,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP05MOAS-GL559_Deploy.csv:31]: Looks good.

File: deployment/CP05MOAS-PG564_Deploy.csv [hunk 2]
```diff
@@ -26,4 +26,11 @@ TS20210903,,WJ20211020,,CP05MOAS-PG564-02-DOSTAM000,4,1,2021-09-03T19:45:00,2021
...
+TS20220822,,,,CP05MOAS-PG564-00-ENG000000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGCON-GLDRCP-00564,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-01-CTDGVM000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09307,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-02-DOSTAM000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00170,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-03-FLORTM000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-04093,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-04-FLORTO000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01348,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-05-NUTNRM000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00660,40.13205,-70.74147,,200,200,CNPM/ISPM,,
+TS20220822,,,,CP05MOAS-PG564-06-PARADM000,5,1,2022-08-22T20:08:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50218,40.13205,-70.74147,,200,200,CNPM/ISPM,,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply deployment/CP05MOAS-PG564_Deploy.csv:36]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00660__20220811.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...273.42, 274.23, 275.03, 275.83, 276.63, 277.44, 278.24, 279.04, 279.84, 280.65, 281.45, 282.25, 283.05, 283.85, 284.66, 285.46, 286.26, 287.06, 287.87, 288.67, 289.47, 290.27, 291.08, 291.88, 292.68, 293.48, 294.29, 295.09, 295.89, 296.69, 297.49, 298.3, 299.1, 299.9, 300.7, 301.51, 302.31, 303.11, 303.91, 304.71, 305.51, 306.32, 307.12, 307.92, 308.72, 309.52, 310.32, 311.13, 311.93, 312.73, 313.53, 314.33, 315.13, 315.93, 316.73, 317.53, 318.34, 319.14, 319.94, 320.74, 321.54, 322.34, 323.14, 323.94, 324.74, 325.54, 326.34, 327.14, 327.94, 328.74, 329.54, 330.34, 331.14, 331.94, 332.74, 333.53, 334.33, 335.13, 335.93, 336.73, 337.53, 338.33, 339.12, 339.92, 340.72, 341.52, 342.32, 343.11, 343.91, 344.71, 345.51, 346.3, 347.1, 347.9, 348.69, 349.49, 350.29, 351.08, 351.88, 352.67, 353.47, 354.26, 355.06, 355.85, 356.65, 357.44, 358.24, 359.03, 359.83, 360.62, 361.42, 362.21, 363, 363.8, 364.59, 365.38, 366.18, 366.97, 367.76, 368.56, 369.35, 370.14, 370.93, 371.72, 372.51, 373.31, 374.1, 374.89, 375.68, 376.47, 377.26, 378.05, 378.84, 379.63, 380.42, 381.21, 382, 382.78, 383.57, 384.36, 385.15, 385.94, 386.72, 387.51, 388.3, 389.09, 389.87, 390.66, 391.44, 392.23, 393.02, 393.8]",
```
Comment (snwhite) [inline calibration/NUTNRM/CGINS-NUTNRM-00660__20220811.csv:8]: Looks good. Spot checked.

**Review by snwhite — APPROVED** [review]

File: calibration/NUTNRM/CGINS-NUTNRM-00660__20220811.csv (hunk 3 above)
Comment (sbeaulieu) [inline calibration/NUTNRM/CGINS-NUTNRM-00660__20220811.csv:8]: Looks good checked using EXCEL

**Review by sbeaulieu — APPROVED** [review]
I commented on and approve the NUTNRM. I did not comment on the other 2 csv files.

**Pull request merged**